docs/CONTRIBUTING.md

```
@@ -1,37 +1,94 @@
+This creates a copy under your GitHub account.
+
+---
+
+### 2. Clone Your Fork
+
+```bash
+git clone https://github.com/your-username/mindjunkies.git
+cd mindjunkies
+```
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)